code/lib/cli-storybook/src/codemod/helpers/story-to-csf-factory.ts

```diff
@@ -1,11 +1,25 @@
 /* eslint-disable no-underscore-dangle */
-import { types as t } from 'storybook/internal/babel';
+import { types as t, traverse } from 'storybook/internal/babel';
 import { isValidPreviewPath, loadCsf, printCsf } from 'storybook/internal/csf-tools';
 
 import type { FileInfo } from '../../automigrate/codemod';
 import { logger } from '../csf-factories';
 import { cleanupTypeImports } from './csf-factories-utils';
 
+// Name of properties that should not be renamed to `Story.input.xyz`
+const reuseDisallowList = ['play', 'parameters', 'run', 'extends', 'render'];
```

Comment:
@kasperpeulen please confirm

Reply:
Isn't it `['args', 'globals', 'run', 'extends']`?

Reply:
There were so many discussions that I lost track, so I'll wait for confirmation

Reply:
Fine. 😭